xenit/solr6/local/93-restore-from-backup.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -e
+
+echo "Solr restore from backup start"
+
+if [ -z ${RESTORE_FROM_BACKUP} ]
+then
+    echo "The index should not be restored from backup, exiting"
+else
+    # if data folder is empty restore from backup
+    if [ -d "${SOLR_DATA_ROOT}/index/alfresco/index" -a `du -s "${SOLR_DATA_ROOT}/index/alfresco/index" | cut -f 1` -gt 100 ]
+    then
+        echo "Index folder existing, skipping backup restore"
+        ls -ltr "${SOLR_DATA_ROOT}/index/alfresco/index"
+    else
+        setOption enable.alfresco.tracking false "${SOLR_DIR_ROOT}/alfresco/conf/solrcore.properties"
+        echo "*************** starting solr without tracking **************************"
+        gosu "${user}" "${SOLR_INSTALL_HOME}/solr/bin/solr" start -m "${JAVA_XMX}" -p "${PORT}" -h "${SOLR_HOST}" -s "${SOLR_DIR_ROOT}" -a "${JAVA_OPTS}"
+        sleep 30
+        echo "*************** solr without tracking started **************************"
+        # get the name of latest snapshot to be restored from the backup
+        if [ -z ${RESTORE_BACKUP_NAME} ]
+        then
+            RESTORE_BACKUP_NAME=`curl -s -L -XGET -u ${BACKUP_USERNAME}:${BACKUP_PASSWORD} "http://${BACKUP_ENDPOINT}/${BACKUP_BUCKET}/success?domain=${BACKUP_DOMAIN}" | tail -1 | cut -d '.' -f 2`
+        fi
+        echo "RESTORE_BACKUP_NAME=${RESTORE_BACKUP_NAME}"
+        if [[ ! ${RESTORE_BACKUP_NAME} == "20"* ]]
```

Comment:
Where does this `"20"*` come from? Is that trying to match specific backup files that start with the current year?